core/matrix/coo.cpp

```diff
@@ -143,6 +146,123 @@ void Coo<ValueType, IndexType>::move_to(
 }
 
 
+/*
+template <typename ValueType, typename IndexType>
+inline size_type mem_size_bccoo(std::shared_ptr<const ReferenceExecutor> exec,
+                                const IndexType *row_idxs,
+                                const IndexType *col_idxs,
+                                const size_type num_rows,
+                                IndexType rows, IndexType offsets,
+                                const size_type block_size,
+                                                                                                                                size_type *mem_size) // GKO_NOT_IMPLEMENTED;
```

Comment:
The majority of added content of this file is commented code, which should be removed.

Reply:
Done